src/WebJobs.Script/extensionrequirements.json

```diff
@@ -0,0 +1,68 @@
+﻿// see type ExtensionStartupTypeRequirement for notes on how these settings are used.
+{
+  "bundles": [
+    {
+      "id": "Microsoft.Azure.Functions.ExtensionBundle",
+      "minimumVersion": "2.7.0"
+    }
+  ],
+  "types": [
+    {
+      "name": "AzureStorageWebJobsStartup",
```

Comment:
Wondering if it would be a good idea to update all the extensions (create issues) with a comment on their startup type letting folks know they shouldn't be renamed  🤔

Reply:
Since we are just protecting against older extensions, I am not too worried about this. Yes this implementation would get more complicated if the types were renamed and we were doing similar enforcement in another major version (say V5).. but if that does end up becoming an issue, there are other strategies we could use - we could ignore the startup type and just use its assembly, for example. I went with basing it on startup types because it does provide a nice workaround if we ever need to circumvent this behavior...

Reply:
Make sense.